db/schema.rb

```diff
@@ -339,8 +339,8 @@
     t.citext "support_identifier", null: false
     t.boolean "is_test"
     t.integer "school_type", default: 0, null: false
-    t.index "lower((first_name)::text)", name: "index_users_on_first_name"
-    t.index "lower((last_name)::text)", name: "index_users_on_last_name"
+    t.index "lower((first_name)::text)", name: "index_users_on_lower_first_name"
+    t.index "lower((last_name)::text)", name: "index_users_on_lower_last_name"
```

Comment:
As much as I prefer the new name, I think you should force the index to the old name to prevent issues if we delete/recreate these indices in the future.

Reply:
Solved.